### ISSUE TITLE
Easily Disable Chart Animations

### ISSUE THREAD

**Issue by rjusro** [swimlane/ngx-charts#229]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```
**What is the motivation / use case for changing the behavior?**
It would be very useful to have a quick-option to turn animations on/off -- mostly for the sake of performance.

```
[animation] = "animation"

//options
animation =  false;
```

This would help in situations where performance may be a concern, or if a chart is being used multiple times on the same view/page. For example, I am using a normalized horizontal bar chart in a table column/cell (one per row) to represent percentages of the relevant data, and 25 rows of them animating at the same time is bit much. I realize this may be a unique use-case; however, turning animations off may, in some cases, be just as important as their glamour. 

If there is already an easy way to turn animations off, it is not currently documented.

:)

**Comment by marjan-georgiev** [comment]
This is a valid feature request. We are waiting for the new animation features to land in angular so we can improve our animations. I agree there should be an option to disable them.

**Comment by VecchioIdraulico** [comment]
I'd also like to see a way to disable the data-point behaviours in the Angular 2 branch, if that's feasible.

**Comment by brent-williams** [comment]
Attached shows attempted use of gauge.  If animation cannot be disabled that pretty much makes e.g. the gauge unusable, as on every update the gauge redraws from position 0.  Even a quick hack to disable animations might allow us to proceed w/ ngx-charts, please advise
![may-17-2017 21-36-36](https://cloud.githubusercontent.com/assets/1694687/26172676/87a85918-3b49-11e7-9036-bde83c8647af.gif)
.

**Comment by jsheldon** [comment]
I could use this change as well.  Here's an updated Plnkr of the demo to show what @brent-williams was describing.

https://plnkr.co/edit/MylIDCFIpzg1oH5tatDs?p=preview

**Comment by michaelfaisst** [comment]
Any updates on this?

**Comment by ItsEcholot** [comment]
This really desperately needs a solution... The gauge element is unusable with updating values.

**Comment by mhmmxx** [comment]
Is there a fix in the works for this?

**Comment by philjones88** [comment]
Any updates on this?

**Comment by marjan-georgiev** [comment]
The gauge animation issue is addressed by https://github.com/swimlane/ngx-charts/pull/563

I am working on adding `animations` input to the charts that would disable the animations for the component.

**Comment by marjan-georgiev** [comment]
`animations` input landed in 6.1.0

**Comment by khaukheng** [comment]
![image](https://user-images.githubusercontent.com/17702727/34234505-6c6ab8fc-e626-11e7-8912-be8ea64ca4bd.png)
Hi, After disable the animation, the pie grid chart show two % , how to solve this? Thanks
